src/components/molecules/NavBar/NavBar.scss

```diff
@@ -270,3 +270,16 @@ $border-radius: 28px;
     display: none;
   }
 }
+
+.nav-sparkle-back-home-icon {
+  width: 25px;
+  height: 25px;
+  background-image: url(/assets/icons/home.png);
```

Comment:
Please, use an icon from `@fortawesome/free-solid-svg-icons` lib. It's as easy as 

`import {
  faChevronRight,
  faChevronLeft,
  faCommentDots,
} from "@fortawesome/free-solid-svg-icons";

<FontAwesomeIcon icon={faChevronRight} size="sm" />
`

Reply:
fixed